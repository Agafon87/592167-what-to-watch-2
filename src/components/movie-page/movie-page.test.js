import React from "react";
import renderer from "react-test-renderer";
import MoviePage from "../movie-page/movie-page.jsx";

it(`MovePage correctly renders after test`, () => {
  const tree = renderer
    .create(<MoviePage />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
