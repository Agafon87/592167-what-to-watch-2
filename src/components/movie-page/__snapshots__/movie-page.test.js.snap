// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePage component renders correctly 1`] = `
Array [
  <section
    className="movie-card movie-card--full"
  >
    <div
      className="movie-card__hero"
    >
      <div
        className="movie-card__bg"
      >
        <img
          alt="Авиатор"
        />
      </div>
      <h1
        className="visually-hidden"
      >
        WTW
      </h1>
      <header
        className="page-header movie-card__head"
      >
        <div
          className="logo"
        >
          <a
            className="logo__link"
            href="/"
            onClick={[Function]}
          >
            <span
              className="logo__letter logo__letter--1"
            >
              W
            </span>
            <span
              className="logo__letter logo__letter--2"
            >
              T
            </span>
            <span
              className="logo__letter logo__letter--3"
            >
              W
            </span>
          </a>
        </div>
        <div
          className="user-block"
        >
          <a
            className="user-block__link"
            href="/login"
            onClick={[Function]}
          >
            Sign in
          </a>
        </div>
      </header>
      <div
        className="movie-card__wrap"
      >
        <div
          className="movie-card__desc"
        >
          <h2
            className="movie-card__title"
          >
            Авиатор
          </h2>
          <p
            className="movie-card__meta"
          >
            <span
              className="movie-card__genre"
            >
              Dramas
            </span>
            <span
              className="movie-card__year"
            >
              2004
            </span>
          </p>
          <div
            className="movie-card__buttons"
          >
            <button
              className="btn btn--play movie-card__button"
              onClick={[MockFunction]}
              type="button"
            >
              <svg
                height="19"
                viewBox="0 0 19 19"
                width="19"
              >
                <use
                  xlinkHref="#play-s"
                />
              </svg>
              <span>
                Play
              </span>
            </button>
            <button
              className="btn btn--list movie-card__button"
              onClick={[MockFunction]}
              type="button"
            >
              <svg
                height="20"
                viewBox="0 0 19 20"
                width="19"
              >
                <use
                  xlinkHref="#add"
                />
              </svg>
              <span>
                My list
              </span>
            </button>
            <a
              className="btn movie-card__button"
              href="/films/undefined/review"
              onClick={[Function]}
            >
              Add review
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="movie-card__wrap movie-card__translate-top"
    >
      <div
        className="movie-card__info"
      >
        <div
          className="movie-card__poster movie-card__poster--big"
        >
          <img
            alt="Авиатор"
            height="327"
            width="218"
          />
        </div>
        <div
          className="movie-card__desc"
        >
          <nav
            className="movie-nav movie-card__nav"
          >
            <ul
              className="movie-nav__list"
            >
              <li
                className="movie-nav__item "
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Overview
                </a>
              </li>
              <li
                className="movie-nav__item "
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Details
                </a>
              </li>
              <li
                className="movie-nav__item "
              >
                <a
                  className="movie-nav__link"
                  href="#"
                  onClick={[Function]}
                >
                  Reviews
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </div>
  </section>,
  <div
    className="page-content"
  >
    <section
      className="catalog catalog--like-this"
    >
      <h2
        className="catalog__title"
      >
        More like this
      </h2>
      <div
        className="catalog__movies-list"
      >
        <div
          className="catalog__movies-list"
        >
          <article
            className="small-movie-card catalog__movies-card"
            onMouseEnter={null}
          >
            <a
              href="/films/1"
              onClick={[Function]}
            >
              <video
                className=""
                height="175"
                width="280"
              >
                Your browser does not support the video tag.
              </video>
            </a>
            <h3
              className="small-movie-card__title"
            >
              <a
                className="small-movie-card__link"
                href="/films/1"
                onClick={[Function]}
              >
                Фантастические твари: Преступления Грин-де-Вальда
              </a>
            </h3>
          </article>
          <article
            className="small-movie-card catalog__movies-card"
            onMouseEnter={null}
          >
            <a
              href="/films/2"
              onClick={[Function]}
            >
              <video
                className=""
                height="175"
                width="280"
              >
                Your browser does not support the video tag.
              </video>
            </a>
            <h3
              className="small-movie-card__title"
            >
              <a
                className="small-movie-card__link"
                href="/films/2"
                onClick={[Function]}
              >
                Богемская рапсодия
              </a>
            </h3>
          </article>
        </div>
      </div>
    </section>
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
          href="/"
          onClick={[Function]}
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          © 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
